src/plugins/data_views/server/saved_objects/data_views.ts

```diff
@@ -15,6 +15,7 @@ export const dataViewSavedObjectType: SavedObjectsType = {
   hidden: false,
   namespaceType: 'single',
   management: {
+    displayName: 'Data view',
```

Comment:
@mattkime We missed `Data view` is capitalized and it's displaying an inconsistent list from the categories drop-down in Saved Objects Management:
![Screen Shot 2022-03-09 at 4 02 14 PM](https://user-images.githubusercontent.com/11909450/157553735-68cceefc-3556-4299-8cb2-a67bd37040b6.png)
Is this something we should fix?

Reply:
I guess so. its in src/plugins/data_views/server/saved_objects/data_views.ts if you're working on something. Or do you want me to open a PR?

Reply:
created https://github.com/elastic/kibana/pull/127533